### ISSUE TITLE
PDC adaption

### ISSUE THREAD

**Issue by DarkTabby** [YelloooBlue/HKUST-iCalendar#3]
Thanks to author's effort, it's really nice and helps me lot.
--------------------------------------------------------------------------
[PDC](https://pdc.hkust-gz.edu.cn/enrollment-records) has simillar pattern, hope to extend it.

**Comment by YelloooBlue** [comment]
It is OK to adapt to the PDC, but I noticed that PDC courses **are often rescheduled**, and if you totally rely on the calendar, you may miss the classes. 
Additionally, PDC System will sends an email before each class to remind you, do you still think it is necessary to adapt to PDC?

**Comment by YelloooBlue** [comment]
> ## Thanks to author's effort, it's really nice and helps me lot.
> [PDC](https://pdc.hkust-gz.edu.cn/enrollment-records) has simillar pattern, hope to extend it.

**Comment by YelloooBlue** [comment]
It is OK to adapt to PDC, but I noticed that PDC courses **are often rescheduled**, and if you totally rely on the calendar, you may miss the classes. 

Additionally, PDC System will sends an email before each class to remind you, do you still think it is necessary to adapt to PDC?

**Comment by DarkTabby** [comment]
That's right, it will be a little inconvenient.

**Comment by YelloooBlue** [comment]
Yep, I think one purpose of adapting to PDC is to check the course conflicts between SIS and PDC? 
May be I will develop a new tool to do this.
